frontend/src/components/Layout/Menu/styled.ts

```diff
@@ -39,19 +39,21 @@ export const MenuTabs = styled('div')({
 export const LeftArrow = styled(leftArrow)({
   display: 'block',
   marginRight: '10px',
+  color: 'black',
 });
 
 export const RightArrow = styled(rightArrow)({
   display: 'block',
   marginRight: '10px',
+  color: 'black',
 });
 
 export const RightArrowBox = styled('div')({
   display: 'flex',
   justifyContent: 'flex-end',
   position: 'absolute',
   right: '12%',
-  top: '1%',
+  top: 'calc(50% - 13px)',
   minWidth: '280px !important',
```

Comment:
Return arrows, and distance for correctly for their on previously position position please.Its work fine, and updated with prev pr.